x-pack/plugins/kuery_autocomplete/index.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { resolve } from 'path';
+
+
+export const kueryAutocomplete = (kibana) => new kibana.Plugin({
+  id: 'kuery_autocomplete',
+  publicDir: resolve(__dirname, 'public'),
+  require: ['kibana', 'xpack_main'],
+  config: (Joi) => Joi.object({
+    enabled: Joi.boolean().default(true)
+  }).default(),
+  uiExports: {
+    hacks: ['plugins/kuery_autocomplete/hacks'],
+  },
+  init: () => {}
```

Comment:
For some reason if I don't include this, the plugin doesn't load...

Reply:
Hmm, works for me, plugins without init functions are nothing new: https://github.com/elastic/kibana/blob/master/src/core_plugins/kbn_vislib_vis_types/index.js. Maybe you mean you're not seeing the log line about its status? That's because plugins without an `init()` function can't actually have a status, since the only way to set it would be with an `init()` function.